examples/images/config.json

```diff
@@ -39,7 +39,7 @@
             ]
         ]
     },
-    "curvature_correction": {
+    "geometry": {
```

Comment:
I wonder whether it would not be actually better to use consistently `curvature_correction`, as we did not change the name of `CurvatureCorrection`.

Reply:
I changed it to curvature to be consistent with drift and color. Will you modify in the analysis scripts so that they by default looks for curvature and not geometry? Or should I just go into analysisbase.py and change it?

Reply:
I can do it.